openmls/src/ciphersuite/secret.rs

```diff
@@ -218,6 +218,11 @@ impl Secret {
     pub(crate) fn ciphersuite(&self) -> &'static Ciphersuite {
         self.ciphersuite
     }
+
+    /// Returns the version of the secret
+    pub(crate) fn version(&self) -> ProtocolVersion {
```

Comment:
I thought we were getting rid of this?

Reply:
I think the way forward is to create an issue for pre-release compatibility and to flag it here.